whats_fresh/whats_fresh_api/views/entry/vendors.py

```diff
@@ -219,7 +220,7 @@ def vendor_list(request):
         'message': message,
         'new_url': reverse('new-vendor'),
         'new_text': "New Vendor",
-        'title': "All Vendors",
+        'title': get_fieldname.get_fieldname('vendors', 'vendors'),
```

Comment:
But why does it take 2 arguments? What does that accomplish over just taking the string we need?